pkg/config/config.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+
+	"gopkg.in/yaml.v2"
+)
+
+type Configurer interface {
+	GetSynonymCategory(category string) string
+}
+
+type Synonym struct {
+	Categories map[string]string
```

Comment:
should be a list of synonyms maybe?

Reply:
in future, but now only support 1 synonym